pkg/neorpc/result/version.go

```diff
@@ -15,12 +15,13 @@ type (
 	// Version model used for reporting server version
 	// info.
 	Version struct {
-		TCPPort   uint16   `json:"tcpport"`
-		WSPort    uint16   `json:"wsport,omitempty"`
-		Nonce     uint32   `json:"nonce"`
-		UserAgent string   `json:"useragent"`
-		Protocol  Protocol `json:"protocol"`
-		RPC       RPC      `json:"rpc"`
+		TCPPort     uint16      `json:"tcpport"`
+		WSPort      uint16      `json:"wsport,omitempty"`
+		Nonce       uint32      `json:"nonce"`
+		UserAgent   string      `json:"useragent"`
+		Protocol    Protocol    `json:"protocol"`
+		Application Application `json:"application"`
```

Comment:
We need to document this as an extension in https://github.com/nspcc-dev/neo-go/blob/master/docs/rpc.md#getversion.